src/interpret.test.ts

```diff
@@ -40,15 +40,24 @@ const results = Convert.toJSONResult(`{
     }
   }`);
 
+const windowsResults = Convert.toJSONResult(`{
+  "#select":{"columns":[
```

Comment:
Not a huge deal because this is test input, but is this crazy indenting / comma positioning intentional?

Reply:
No it wasn't! I've tidied it up now, thanks.